people.json

```diff
@@ -16,5 +16,11 @@
     "github": "@shash137",
     "twitter": "@shash137",
     "linkedin": "https://www.linkedin.com/in/shashank-shukla-4b960a169/"
+  },
+  {
+    "name": "Wesley Peeters",
+    "github": "@aracQuango",
+    "reddit": "u/i-dont-like-gucci",
```

Comment:
There is unfortunately no reddit tag (yet?) but you could create another pull-request and add the functionality? ;)